source/screens/bookingScreens/BookingSummary.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components/native";
+import { BookingItem } from "../../helpers/MockBookingData";
+import icons from "../../helpers/Icons";
+import { Text } from "../../components/atoms";
+import {
+  DateTimeCard,
+  AddressCard,
+  CharacterCard,
+} from "../../components/molecules";
+
+const Container = styled.ScrollView`
+  height: 80%;
+  margin-top: 0px;
+  padding-left: 16px;
+  padding-right: 16px;
+  background-color: ${({ theme }) => theme.colors.neutrals[6]};
+`;
+
+const Title = styled(Text)`
+  margin-top: 50px;
+  margin-bottom: 15px;
+`;
+
+interface BookingSummaryProps {
+  route: any;
+}
+
+const BookingSummary = ({ route }: BookingSummaryProps): JSX.Element => {
+  const routeName = route?.name;
```

Comment:
I cant see that `routeName` is used in more than one place, so perhaps this could be simplified to:

```js
const isConfirmation = route?.name === 'Confirmation'
```